db/migrate/20160114121308_remove_link_fields.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveLinkFields < ActiveRecord::Migration
```

Comment:
That seems fine, though we'd ideally like to ensure we can deploy this smoothly.  We don't have much tagging at all going on at present, though, so as long as we can copy the data over, then deploy this change within a few minutes, I think it will be fine.